src/plugins/piral-feeds/src/withFeed.tsx

```diff
@@ -13,7 +13,7 @@ export function withFeed<TData, TItem, TProps>(
     if (!loaded) {
       return <PiralLoadingIndicator />;
     } else if (data) {
-      return <Component data={data} {...props} />;
+      return <Component {...props} data={data} />;
```

Comment:
What's your take on this syntax:
```jsx
return <Component {...{ ...props, data }} />;
```
I believe this change has been made so data overwrites props. Imho the concatenated syntax makes this more explicit and noticalbe.

Reply:
I think they are equivalent, but the used version is (at least for me) more transparent. It says "the react component should take all the values from props and set data explicitly", while the other version says "the react component should take all the values from an object that takes all the values from props and sets data to data".

Reply:
I read the syntax as: "pass the props and set/overwrite data". This is a question of taste, just wanted to mention it.